client/src/store/login/actions.ts

```diff
@@ -1,11 +1,17 @@
 import API from "../api";
-import { showAlert } from "../../store/alert/actions";
-const TOKEN = "TOKEN";
+import {setToken} from "./setToken.helper";
+
+const TOKEN_WORD_LENGTH = 6;
 
 export const loginUser = (user: object) => (dispatch: Function) => {
   return API.post("/auth/login", user)
     .then(response => {
-      localStorage.setItem(TOKEN, response.data.token);
+        setToken(response.data.token);
     })
     .catch(err => console.log(err.message));
 };
+
+export const cookieToLocalStorage = () => (dispatch: Function): void => {
+    const cookie = document.cookie.slice(TOKEN_WORD_LENGTH);
```

Comment:
why do we need cookie?

Reply:
We can't get response from google with token, because on click user goes to /auth/google, and then google sends parameters to /auth/google/callback and then redirect. We can't pass parameters to /auth/google/callback and get res, because we need google window.